src/index.js

```diff
@@ -80,6 +81,33 @@ class ChluAPIPublish {
       }
     })
 
+    api.post('/crawl', async (req, res) => {
+      try {
+        this.log('POST CRAWL => ...')
+
+        const data = req.body
+
+        console.log(data)
+
+        const crawlerType = data.type
+        const crawlerUrl = data.url
+        const crawlerDid = data.did
```

Comment:
just noticed that on the frontend we send the did __with the private key__. This is really bad, only the did id should be sent